katzen.nix

```diff
@@ -0,0 +1,61 @@
+{ src ? ./., version ? "unstable" }:
+{ lib, buildGoModule, fetchFromGitHub, copyDesktopItems, makeDesktopItem
+, pkg-config, librsvg, libGL, libX11, libXfixes, libxkbcommon, vulkan-headers
+, wayland, xorg }:
+
+buildGoModule rec {
+  pname = "katzen";
+  inherit src version;
+
+  vendorHash = "sha256-LsaCb2NLc8bVLlRk+JQr0ZAaEqsIdH/GLO1wvaVyKrY=";
```

Comment:
What is vendorHash? How can I figure out which branch is being built?

Reply:
> What is vendorHash?

my understanding is that vendorHash is effectively the hash of all of the go deps. so, in the future, after we upgrade any go dependencies (eg do anything that touches go.mod) then our nix build will begin to fail with an error that tells us the new vendorHash. after copy+pasting that new value into `katzen.nix` it will work again.

it would be nice to have a make target that automates this step, but I haven't written that yet.

> How can I figure out which branch is being built?

one can use this packaging to build any branch (which contains the two .nix files) of katzen.